pystiche/image/transforms/crop.py

```diff
@@ -103,7 +103,7 @@ def randint(range: int) -> int:
             return torch.randint(range + 1, ()).item()
 
         vert_origin = randint(image_height - crop_height)
-        horz_origin = randint(image_height - crop_height)
+        horz_origin = randint(image_width - crop_width)
```

Comment:
This was not detected since the test also used `ValidRandomCrop.get_random_origin()` to create the crop origin.